tests/func-tests/defaults_test.go

```diff
@@ -0,0 +1,202 @@
+package tests_test
+
+import (
+	"context"
+	"fmt"
+	"reflect"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/utils/pointer"
+
+	"kubevirt.io/client-go/kubecli"
+
+	"github.com/kubevirt/hyperconverged-cluster-operator/api/v1beta1"
+	tests "github.com/kubevirt/hyperconverged-cluster-operator/tests/func-tests"
+)
+
+const (
+	removePathPatch = `[{"op": "remove", "path": %q}]`
+)
+
+var _ = Describe("Check Default values", Label("defaults"), Serial, func() {
+	var (
+		cli kubecli.KubevirtClient
+		ctx context.Context
+	)
+
+	BeforeEach(func() {
+		var err error
+		cli, err = kubecli.GetKubevirtClient()
+		Expect(cli).ToNot(BeNil())
+		Expect(err).ToNot(HaveOccurred())
+
+		ctx = context.Background()
+	})
+
+	restoreDefaults := func(cli kubecli.KubevirtClient) error {
+		return tests.PatchHCO(ctx, cli, []byte(`[{"op": "replace", "path": "/spec", "value": {}}]`))
+	}
+
+	Context("certConfig defaults", func() {
+		defaultCertConfig := v1beta1.HyperConvergedCertConfig{
+			CA: v1beta1.CertRotateConfigCA{
+				Duration:    &metav1.Duration{Duration: time.Hour * 48},
+				RenewBefore: &metav1.Duration{Duration: time.Hour * 24},
+			},
+			Server: v1beta1.CertRotateConfigServer{
+				Duration:    &metav1.Duration{Duration: time.Hour * 24},
+				RenewBefore: &metav1.Duration{Duration: time.Hour * 12},
+			},
+		}
+
+		DescribeTable("Check that certConfig defaults are behaving as expected", func(path string) {
+			Expect(restoreDefaults(cli)).To(Succeed())
+
+			patch := []byte(fmt.Sprintf(removePathPatch, path))
+			Eventually(func(g Gomega) {
+				err := tests.PatchHCO(ctx, cli, patch)
+				g.Expect(err).ShouldNot(HaveOccurred())
```

Comment:
This could be changed into a one statement:
```suggestion
				g.Expect(tests.PatchHCO(ctx, cli, patch)).ShouldNot(HaveOccurred())
```
Again, it's something that depends on personal preference, so this is more of a comment than a request to change.